support/Pipelines/Bbh/InitialData.py

```diff
@@ -76,6 +80,28 @@ def generate_id(
         " generated input files."
     )
 
+    # Resolve directories
+    if pipeline_dir:
+        pipeline_dir = Path(pipeline_dir).resolve()
+    assert segments_dir is None, (
+        "Initial data generation doesn't use segments at the moment. Specify"
+        " '--run-dir' / '-o' or '--pipeline-dir' / '-d' instead."
+    )
+    if evolve:
+        assert pipeline_dir is not None, (
+            "Specify a '--pipeline-dir' / '-d' to evolve the initial data."
+            " Don't specify a '--run-dir' / '-o' because it will be created in"
+            " the 'pipeline_dir' automatically."
+        )
+        assert run_dir is None, (
+            "Specify the '--pipeline-dir' / '-d' rather than '--run-dir' / '-o'"
+            " when evolving the initial data. Directories for the initial data,"
+            " evolution, etc will be created in the 'pipeline_dir'"
+            " automatically."
+        )
+    if pipeline_dir and not run_dir:
+        run_dir = pipeline_dir / "001_ID"
```

Comment:
One index or zero index?

Also, not for right now, but once we have eccentricity reduction, we're going to maybe have to change the numbering since we'll need to have multiple ID/Ev rounds

Reply:
Yep this is just a preliminary hardcoded numbering from 1 to 3. With ecc control etc we can make this dynamic and just increment the number with each ecc step.